cli/jobs/pipelines-with-components/flow_in_pipeline/1a_flow_in_pipeline/pipeline.yml

```diff
@@ -0,0 +1,39 @@
+type: pipeline
+
+name: simplePipelineJobWithFlow
+description: The hello world pipeline job with flow option 1
```

Comment:
what's option 1?

Reply:
it's copied from previous sample in azure-ai-ml, should be removed